<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="dPLAP166" failures="0" tests="1" name="XGLT_15177_Test" time="3.766" errors="1" timestamp="2023-08-08T14:16:15 IST" skipped="0">
  <testcase classname="XGLT_15177_Test" name="About" time="3.766">
    <error message="Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.pass(com.aventstack.extentreports.model.Media)&quot; because the return value of &quot;java.lang.ThreadLocal.get()&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentTest.pass(com.aventstack.extentreports.model.Media)" because the return value of "java.lang.ThreadLocal.get()" is null
at helper.ScreenshotUtility.onDemandScreenshotReport(ScreenshotUtility.java:10)
at XGLT_15177_Test.About(XGLT_15177_Test.java:41)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at TestngRunner.main(TestngRunner.java:9)
]]>
    </error>
  </testcase> <!-- About -->
  <system-out>
    <![CDATA[InstanceQA134-PS xGL DB
InstanceQA134-PS xGL DBData Service10.7.30.164:1521/xgldb
InstanceQA134-PS xGL DBData Service10.7.30.164:1521/xgldbChannel CountChannelCount: 560, HeadendCount=77
InstanceQA134-PS xGL DBData Service10.7.30.164:1521/xgldbChannel CountChannelCount: 560, HeadendCount=77Build Infodevelop | ID=2777abc 84 @ 08/07/23 06:35 AM
InstanceQA134-PS xGL DBData Service10.7.30.164:1521/xgldbChannel CountChannelCount: 560, HeadendCount=77Build Infodevelop | ID=2777abc 84 @ 08/07/23 06:35 AMPackage InfoxGLinear 2.6.0 Azure Build at 08/07/2023 06:40:58]]>
  </system-out>
</testsuite> <!-- XGLT_15177_Test -->
